constants.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  ERC20_ADAPTER: '0xD1b8592EFCA1613c837F408419BBF0e4B85142D8',
+  ERC20_ADAPTER: '0x392B33359e4fa56cCf7897e753d8f04Eb7d130a8',
```

Comment:
let's not change the existing ERC20_ADAPTER. we should add this as `ERC20_ADAPTER_02`